software/scripts/temporal_gain_and_phase_mism.jl

```diff
@@ -11,12 +13,17 @@ function eval_gain_and_phase_mism(;
     frequency = 1575.42u"MHz",
     sample_rate = 6u"MHz",
     baseband_frequency = 0.6u"MHz",
-#    gain = 60u"dB",
+    run_time = 10u"s"
 )
 
-    Device(first(Devices())) do dev
+    device_kwargs = Dict{Symbol,Any}()
+    if chomp(String(read(`hostname`))) == "pathfinder"
+        device_kwargs[:driver] = "XTRX"
+        device_kwargs[:serial] = "12cc5241b88485c"
+    end
```

Comment:
Not that this is bad, but IMO it's probably better to enumerate devices, and if this serial exists, use it, otherwise take the first available. :)

Also, you might want to unconditionalyl set `device_kwargs[:driver] = "XTRX"